client/vitest.setup.ts

```diff
@@ -0,0 +1 @@
+import 'vitest-canvas-mock';
```

Comment:
is there a way to avoid needing this extra file?

Reply:
no - it provides the capability to extract the imageData from the imageBitmap that comes out of fftToRGB, so it can be compared to the expected outcome in tests.

Reply:
though maybe as a dev dependency?

Reply:
just confirmed - it's only installed during test.

Reply:
sorry i meant the file itself, is there a way to include it in vite.config.ts without needing a whole separate file?

Reply:
We might change the place so it is not another file in root.

Reply:
removed the file. we'll see if it's really needed.